templates/page/page.tpl.php

```diff
@@ -0,0 +1,149 @@
+<?php
+/**
+ * @file
+ * U.S. Web Design Standards theme implementation to display the default Drupal
+ * page. Credit goes to the Bartik team for this awesome documentation!
+ *
+ * The doctype, html, head and body tags are not in this template. Instead they
+ * can be found in the html.tpl.php template normally located in the directory
+ * above.
+ *
+ * Available variables:
+ *
+ * General utility variables:
+ * - $base_path: The base URL path of the Drupal installation. At the very
+ *   least, this will always default to /.
+ * - $directory: The directory the template is located in, e.g. modules/system
+ *   or themes/bartik.
+ * - $is_front: TRUE if the current page is the front page.
+ * - $logged_in: TRUE if the user is registered and signed in.
+ * - $is_admin: TRUE if the user has permission to access administration pages.
+ *
+ * Site identity:
+ * - $front_page: The URL of the front page. Use this instead of $base_path,
+ *   when linking to the front page. This includes the language domain or
+ *   prefix.
+ * - $logo: The path to the logo image, as defined in theme configuration.
+ * - $site_name: The name of the site, empty when display has been disabled
+ *   in theme settings.
+ * - $site_slogan: The slogan of the site, empty when display has been disabled
+ *   in theme settings.
+ * - $hide_site_name: TRUE if the site name has been toggled off on the theme
```

Comment:
Thanks for including this documentation of variables!

I notice that several of them aren't yet referenced in the code. I'm most concerned about ones that users' actions set, like `$hide_site_slogan`, as leaving them out risks not respecting users' settings. Will merge this as-is but make a follow-up ticket for that so it doesn't get lost.